src/llvm/mod.rs

```diff
@@ -8,24 +8,17 @@ use std::{
     ptr, slice,
 };
 
+use iter::{IterModuleFunctions, IterModuleGlobalAliases, IterModuleGlobals};
```

Comment:
weird that this is grouped with external and not crate

Reply:
Probably because it's not using `crate::` and it's not smart enough to figure out that it's inside the crate.

I will just add `crate::` for now.

Reply:
I added `crate::llvm` as a prefix for now, but I guess it's a bug in rustfmt.

Reply:
I guess it's an unstable feature. I think I'd prefer to omit the prefix where it isn't needed.